data_write/data_write.py

```diff
@@ -600,23 +789,33 @@ def write_bfiq_params(parameters_holder):
                 parameters['antenna_arrays_order'] = []
 
                 flattened_data = []
+                num_antenna_arrays = 0
```

Comment:
Can you remind me what happens if we have neither the 'main' or 'intf' flags set for a slice? (I.e. num_antenna_arrays stays at 0)

Reply:
there will always be main, changed so there is no if